types/plugin.d.ts

```diff
@@ -129,11 +129,12 @@ declare namespace fastifyCookie {
     sameSite?: 'lax' | 'none' | 'strict' | boolean;
     /** One of the `Priority` string attributes (`low`, `medium` or `high`) specifying a retention priority for HTTP cookies that will be respected by user agents during cookie eviction. */
     priority?: 'low' | 'medium' | 'high';
-    /** The `boolean` value of the `Secure` attribute. Set this option to false when communicating over an unencrypted (HTTP) connection. Value can be set to `auto`; in this case the `Secure` attribute will be set to false for HTTP request, in case of HTTPS it will be set to true. */
+    /** Add the `Secure` attribute. Defaults to `false`. */
```

Comment:
How does this improve it?

Reply:
`"auto"` is only available in `CookieSerializeOptions`, so it's moved there. 👀